docs/configuration/defining-playbooks/external-playbook-repositories.rst

```diff
@@ -40,12 +40,10 @@ This method works for public repos too:
       # add the same playbooks as above, this time over ssh
       my_extra_playbooks:
         url: "git@github.com:robusta-dev/robusta-chaos.git"
-        key: |-
-          -----BEGIN OPENSSH PRIVATE KEY-----
-          ewfrcfsfvC1rZXktdjEAAAAABG5vb.....
-          -----END OPENSSH PRIVATE KEY-----
+        key: LS0tLS1CRUdJTiBPUEVOU1NIIFBSSVZBVEUgS0VZLS0tL
 
-The ``key`` parameter must contain a deployment key with ``read`` access.
+
+The ``key`` parameter must contain a `base64` encoded deployment key with ``read`` access. 
```

Comment:
@pavangudiwada Can you show an example of what's inside the base64? Inside the base64, is it:

```
-----BEGIN OPENSSH PRIVATE KEY-----
ewfrcfsfvC1rZXktdjEAAAAABG5vb.....
-----END OPENSSH PRIVATE KEY-----
```

If so, would be good to explain

Reply:
Updated as mentioned. 
![image](https://github.com/robusta-dev/robusta/assets/25551553/d9613a16-0f9b-4633-9ef8-634882f29bd3)